proto/README.md

```diff
@@ -20,7 +20,36 @@ The canonical Protobuf definitions live in the `proto` folder of the relevant
 release branch of CometBFT. These definitions are published to the [Buf
 registry][buf] for integrators' convenience.
 
-## Why does CometBFT use `tendermint` Protobuf definitions?
+The Protobuf files are organized under two domains: `cometbft` and `tendermint`.
+The `cometbft.*` packages use version suffixes to let application developers
+target versions of the protocols as they have evolved between CometBFT releases.
+
+## Which CometBFT release does each package belong to?
+
+By the 1.0.0 release, the entire set of Protobuf definitions used by CometBFT
+is published in packages suffixed with `.v1`. Earlier revisions of the
+definitions, where they differed, are provided alongside in `.v1beta`_N_
+packages, corresponding to 0.x releases as follows:
+
+| Domain          | 0.34      | 0.37      | 0.38      |
+|-----------------|-----------|-----------|-----------|
+| `abci`          | `v1beta1` | `v1beta2` | `v1beta3` |
+| `blocksync`     |           | `v1beta1` | `v1`      |
+| `consensus`     | `v1beta1` | same      | same      |
+| `crypto`        | `v1`      | same      | same      |
+| `libs/bits`     | `v1`      | same      | same      |
+| `mempool`       | `v1`      | same      | same      |
+| `p2p`           | `v1`      | same      | same      |
+| `privval`       | `v1beta1` | same      | `v1`      |
+| `rpc/grpc`[^1]  | `v1beta1` | `v1beta2` | `v1beta3` |
+| `state`         | `v1beta1` | `v1beta2` | `v1beta3` |
+| `statesync`     | `v1`      | same      | same      |
+| `types`         | `v1beta1` | `v1beta2` | `v1`      |
+| `version`       | `v1`      | same      | same      |
```

Comment:
This table's great, thanks for putting it together! What do the `same` entries mean though?

Also, could you please add a "1.0" column? My assumption is that every field value will be `v1`? Might be good to just have that there to be crystal clear to users as to what to expect.

Reply:
"same" means the definitions have not changed in this release, so the same package as recorded to the left can be used.
Yes, in 1.0 everything is `v1` as the text explains, but I'll add the column as well.